### ISSUE TITLE
Obsolete documentation

### ISSUE THREAD

**Issue by dutch55** [MicrosoftDocs/OfficeDocs-Support#1752]
[Enter feedback here]
Recently Scanpst seems to run more than 90 passes on some PST files, so this documentation is obsolete.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0013ac99-8442-ee88-95d8-c5f459c518e5
* Version Independent ID: 6679377c-f8c6-4e1e-696e-ebf61342e31b
* Content: [Inbox Repair tool (Scanpst.exe) run multiple passes - Outlook](https://docs.microsoft.com/en-us/outlook/troubleshoot/data-files/inbox-repair-tool-scanpst-exe-run-multiple-passes)
* Content Source: [Outlook/Client/data-files/inbox-repair-tool-scanpst-exe-run-multiple-passes.md](https://github.com/MicrosoftDocs/OfficeDocs-Support/blob/public/Outlook/Client/data-files/inbox-repair-tool-scanpst-exe-run-multiple-passes.md)
* Product: **office-perpetual-itpro**
* GitHub Login: @helenclu
* Microsoft Alias: **luche**

**Comment by VLG17** [comment]
@nam31  looking into it.

**Comment by VLG17** [comment]
@dutch55 The documentation states that Scanpst runs a single pass just for Outlook 2016 versions older than 1807. Are you saying that you saw it running multiple passed for these older Outlook versions?

For newer Outlook versions it does run multiple passes, which is also stated in the documentation.

**Comment by dutch55** [comment]
I should have been more precise.

I am referring to scanpst.exe as it is provided with Office 365. I realized in preparing my update, I made an error in my original post. I should have written, "Scanpst seems to run more than 90 phases" instead of referring to passes. 

To summarize my issues:

1. The current scanpst.exe will on at least one .pst file show Phase counts in excess of 100. This isn't documented anywhere I can find.

2. The documentation is inconsistent in use of the terms "passes" and "phases".

3. I have a .pst file that scanpst will make process for a long time with a Phase count in excess of 100.  The file has messages I saved to a .pst from 2020 that have not been changed, but when I run scanpst on the file a second time without opening Outlook, it also runs a lot of passes, so it cannot fix the corruption.

The file was created in two steps,  with no Outlook crashes or disk errors, so it appears that the copy process is causing the corruption. First I manually selected the messages between Jan 1 2020 and December 31 2020, and copied them to a new folder in my local Exchange copy. Then I created an Outlook .pst file, and copied the folder to that .pst file. Then I ran scanpst on that folder. 

4. The table below under my heading "SECOND REFERENCE:"  implies a maximum phase count of 24.  I have seen the count go above 100. from recollection, at least 144, As I post this, I am running one on the aforementioned .pst file thatsays Phase 48 of 48.

DETAILS:

This seems to be the source of truth, but see comments on documentation below:

Inbox Repair tool (Scanpst.exe) run multiple passes ...
https://docs.microsoft.com/en-us/outlook/troubleshoot/data-files/inbox-repair-tool-scanpst-exe-run-multiple-passes

In the description it says some confusing and contradictory things. Note that the word "pass"  seems to have  two different meanings. First, it says the maximum number of passes is 20, but in the description of the command line arguments, it uses the term "scan or repair iterations), but in the second reference on the page, it implies there are three "passes" a Initial, Second, and Third pass.

Then in the description of the -rescan argument, it  uses the term "scan or repair iterations", and says any integer greater than 20, it defaults to 20, leaving the reader to guess if this is the same maximum of 20 used in reference to the term "passes" earlier.

Here are the details, quoting from the above article:

FIRST REFERENCE:

The steady state is reached when one of the following is true:

Zero (0) errors are returned
No new errors are reported (two consecutive passes output the exact same result)
A maximum number of passes is reached (to avoid a possible loop); currently, the limit is 20 (this value defaults to 10 if it's not specified)

SECOND REFERENCE:
Initial pass: Phases 1 to 8
Second pass: Phases 9 to 16
Third pass: Phases 17 to 24

THIRD REFERENCE IN THE DESCRIPTION OF THE OPTIONAL ARGUMENTS:

rescan	Maximum number of scan or repair iterations before reaching steady state	0-20; any integer greater than 20 defaults to 20.

MORE ON INCOMPLETE HARD TO FIND DOCUMENTATION:

The documentation is incomplete, and hard to find with a web search, which makes it hard for a user to figure out what to use:

A Google search of: scanpst.exe returns these entries as  the first.  Bing returns a non-Microsoft URL as the first, and this is the second entry returned.

The first is undated, but says it applies to Office 2019 and Office 365, which doesn't say anything about running multiple passes using command line arguments.
Repair Outlook Data Files (.pst and .ost) - Microsoft Support

https://support.microsoft.com/en-us/office/repair-outlook-data-files-pst-and-ost-25663bc3-11ec-4412-86c4-60458afc5253

The second is dated 04/07/2021, but doesn't say it applies to Office 2016 or Office 365, and doesn't say anything about command line arguments either.
How to repair your Outlook personal folder file (.pst) - Microsoft ..

https://docs.microsoft.com/en-us/outlook/troubleshoot/data-files/how-to-repair-personal-folder-file

Google doesn't return any more Microsoft URLs on the  first page of search results, but Bing returns this one several entries down, dated 04/09/2021.  This one only refers to Outlook 2016 in the title, but in the sub-heading it says it applies to Office 2016 and Office 365. No mention of Office 2019. The text says it is applicable starting with Outlook 2016 version 1807.
Inbox Repair tool (scanpst.exe) run multiple passes ...
https://docs.microsoft.com/en-us/outlook/troubleshoot/data-files/inbox-repair-tool-scanpst-exe-run-multiple-passes
